packages/tupaia-web/src/features/Map/Map.tsx

```diff
@@ -61,7 +61,7 @@ const StyledMap = styled(LeafletMap)`
     top: -50px;
     right: 3px;
     a {
-      background: ${({ theme }) => theme.navigationBtn.main};
+      background: #34353f;
```

Comment:
This theme var was used in just this place, so it made sense to leave it contained in the component